src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -27,8 +27,8 @@
  */
 final class JavaScriptImportPathCompiler implements AssetCompilerInterface
 {
-    // https://regex101.com/r/fquriB/1
-    private const IMPORT_PATTERN = '/(?:import\s*(?:(?:\*\s*as\s+\w+|[\w\s{},*]+)\s*from\s*)?|\bimport\()\s*[\'"`](\.\/[^\'"`]+|(\.\.\/)*[^\'"`]+)[\'"`]\s*[;\)]?/m';
+    // https://regex101.com/r/qFoeoR/1
+    private const IMPORT_PATTERN = '/(?:\'(?:[^\'\\\\]|\\\\.)*\'|"(?:[^"\\\\]|\\\\.)*")|(?:import\s*(?:(?:\*\s*as\s+\w+|[\w\s{},*]+)\s*from\s*)?|\bimport\()\s*[\'"`](\.\/[^\'"`]+|(\.\.\/)*[^\'"`]+)[\'"`]\s*[;\)]?/m';
```

Comment:
Can you add a test case that the `\\\\` is needed for (which is the `\` escape character inside a string)?